tests/rptest/services/redpanda_installer.py

```diff
@@ -591,9 +591,9 @@ def upgrade_path_to_head(
             if line not in latest_version_by_line or v > latest_version_by_line[line]:
                 latest_version_by_line[line] = v
         latest_versions = list(latest_version_by_line.values())
-        latest_versions.sort()
         if self.head_version() not in latest_versions:
```

Comment:
So this is the check that breaks if there is a rc tag? I am trying to understand how the rc tag breaks this